src/main/java/org/opentripplanner/routing/core/routing_parametrizations/RoutingDelays.java

```diff
@@ -0,0 +1,75 @@
+package org.opentripplanner.routing.core.routing_parametrizations;
+
+import org.opentripplanner.routing.core.vehicle_sharing.VehicleDescription;
+
+/**
+ * Describes how long specified actions take.
+ */
+public class RoutingDelays {
+
+    private int kickScooterRentingTime = 30;
+
+    private int kickScooterDropoffTime = 30;
+
+    private int motorbikeRentingTime = 60;
+
+    private int motorbikeDropoffTime = 60;
+
+    private int carRentingTime = 90;
+
+    public int getCarDropoffTime() {
+        return carDropoffTime;
+    }
+
+    private int carDropoffTime = 240;
+
+    public int getRentingTime(VehicleDescription vehicleDescription) {
+        switch (vehicleDescription.getVehicleType()) {
+            case CAR:
+                return carRentingTime;
+            case MOTORBIKE:
+                return motorbikeRentingTime;
+            case KICKSCOOTER:
+                return kickScooterRentingTime;
+            default:
+                return 0;
```

Comment:
To powinno rzucać wyjątkiem, żebyśmy nie zapomnieli dodać czasów wsiadania/wysiadania dla nowych typów pojazdów.

Reply:
Done